### PR TITLE
Add SSH keys for cluster templates

### PR DESCRIPTION
**Pull request by maciaszczykm**
### What this PR does / why we need it
Zrzut ekranu 2021-09-7 o 08 35 14
Zrzut ekranu 2021-09-7 o 08 34 51

### Which issue(s) this PR fixes
Closes https://github.com/kubermatic/dashboard/issues/3634.

### Release note

```release-note
NONE
```

### CODE REVIEW

**Comment by maciaszczykm** [comment]
PTAL @kgroschoff @floreks

File: src/app/shared/components/save-cluster-template/template.html [hunk 1]
```diff
@@ -55,6 +55,12 @@
       </mat-radio-button>
     </mat-radio-group>
   </form>
+  <div *ngIf="showSSHKeyWarning"
+       class="km-dialog-warning"
+       fxLayoutAlign=" center">
+    <i class="km-icon-error"></i>
+    <div>SSH keys will not be saved in the cluster templates that are not in the project scope.</div>
```
Comment (kgroschoff) [inline src/app/shared/components/save-cluster-template/template.html:62]: Not an expert here, but I think the `the` befor cluster templates could be deleted. 🤔 
`SSH keys will not be saved in cluster templates that are not in the project scope.`

Comment (kgroschoff) [reply src/app/shared/components/save-cluster-template/template.html:62]: Other than that no other remarks from my side.

Comment (maciaszczykm) [reply src/app/shared/components/save-cluster-template/template.html:62]: You're right. Fixed.

**Review by kgroschoff — COMMENTED** [review]
/lgtm

**Pull request merged**